code/part-two/client/source/services/requests.js

```diff
@@ -83,7 +84,7 @@ export const getCollections = (key = null) => {
  *   { owner: string } - an object with an owner returns moji owned by that key
  *   { owner: string, dna: string } -
  */
-export const getMoji = (filterOrAddress = null) => {
+export const getMojis = (filterOrAddress = null) => {
```

Comment:
As confusing as the pluralization of of "cryptomoji" is (i.e. it its own plural, like sheep or moose), I think consistency is important here. If we're going to decide that it is pluralized with an s (which I'm not necessarily opposed to), it needs to be changed everywhere, not just in this method name.